### PR TITLE
Supporting both init and systemd services in Linux

### PR DESCRIPTION
**Pull request by rcabral-splunk**
systemd is available in CLI's boot-start command as of v7.2.2 only. We need to be able to support both systemd and init to manage splunkd in Linux.

### CODE REVIEW

**Comment by SkyKnightSKS** [comment]
The play would need adaptations for Universal Forwarder Support.

File: roles/splunk_common/files/Splunkd.service [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[Unit]
+After=network.target
+
+[Service]
+Type=simple
+Restart=always
+ExecStart=/opt/splunk/bin/splunk _internal_launch_under_systemd
```
Comment (nwang92) [inline roles/splunk_common/files/Splunkd.service]: Let's turn this into a Jinja template so we can accodomate variables + support for UF

**Comment by nwang92** [comment]
Can you resolve the merge conflicts here?

File: roles/splunk_common/files/Splunkd.service (hunk 1 above)
Comment (rcabral-splunk) [reply roles/splunk_common/files/Splunkd.service]: In a separate PR perhaps?

**Pull request merged**